### PR TITLE
ace: mtl: config: enabling power domains

### PR DESCRIPTION
**Pull request by tmleman**
Enabling power domains on Meteorlake.

Dependencies:

- [x] https://github.com/thesofproject/sof/pull/6752

### CODE REVIEW

**Review by lgirdwood — COMMENTED** [review]
LGTM, I assume pending validation ?

**Comment by tmleman** [comment]
I haven't found any regression against the current state of the main branch.

**Review by lgirdwood — APPROVED** [review]

**Comment by greg-intel** [comment]
SOFCI TEST

File: west.yml [hunk 1]
```diff
@@ -43,7 +43,7 @@ manifest:
 
     - name: zephyr
       repo-path: zephyr
-      revision: ffed8388e7cb27ac9331337105ee55cc2f2f0a72
+      revision: 56284d7017dc593f18c3aaa128d2c4e596a8e4c8
```
Comment (lyakh) [inline west.yml:46]: this seems important enough to me to deserve a separate PR

**Review by kv2019i — APPROVED** [review]

**Review by lgirdwood — APPROVED** [review]

**Comment by lgirdwood** [comment]
SOFCI TEST

**Comment by lgirdwood** [comment]
Re-runing CI with west update merged.

**Review by pblaszko — APPROVED** [review]

**Review by marcinszkudlinski — APPROVED** [review]

**Comment by lgirdwood** [comment]
@wszypelt are we good to merge ? CI seems stuck ?

**Comment by wszypelt** [comment]
@lgirdwood I restarted the internal intel CI system, within 40 minutes should be up-to-date results

**Comment by abonislawski** [comment]
One fail in Internal CI tests, fw crashed while powering on core 2

@tmleman can you check?

**Comment by wszypelt** [comment]
@abonislawski @tmleman 
It seems that we have a problem with dut and some tests fail, I'm re-testing internal intel CI

**Review by abonislawski — APPROVED** [review]

**Comment by marcinszkudlinski** [comment]
@lgirdwood @mwasko pls merge

**Review by aborisovich — APPROVED** [review]
Lets merge
